qubes-rpc/qubes.Filecopy

```diff
@@ -1,2 +1,6 @@
 #!/bin/sh
+if [ -f /usr/share/nautilus-python/extensions/qvm_nautilus_bookmark.sh ]
+then
+  /bin/sh /usr/share/nautilus-python/extensions/qvm_nautilus_bookmark.sh >/dev/null 2>&1
```

Comment:
Please execute it directly, it has already `#!/bin/sh`.
Also, please connect stdin to `/dev/null` too.